reana/reana_dev/run.py

```diff
@@ -476,12 +477,41 @@ def run_example(
 
     display_message(
         "Running the following matrix:\n"
-        "Demos: {}\n"
-        "Workflow engines: {}\n"
-        "Compute backends: {}".format(
-            ",".join(components), ",".join(workflow_engines), ",".join(compute_backends)
-        )
+        f"Demos: {','.join(components)}\n"
+        f"Workflow engines: {','.join(workflow_engines)}\n"
+        f"Compute backends: {','.join(compute_backends)}"
     )
```

Comment:
Cosmetics: Please use `display_mesage(..., companent='reana')` so that the output would not have empty leading component.  Also, you can print "Demos" etc via the same list as the output, so that the "beginning info style" and "finishing info style" would resemble more closely.  

IOW, currently we have:

```
[2022-04-27T11:00:58] : Running the following matrix:
Demos: reana-demo-root6-roofit
Workflow engines: serial,yadage,snakemake,cwl
Compute backends: kubernetes
```

and after my suggested changes we would have something like:

```
[2022-04-27T11:00:58] reana: reana-dev run-example -b kubernetes -c r-d-r-roofit --check-only
Test matrix:
  - Demo examples: reana-demo-root6-roofit
  - Workflow engines: cwl, serial, snakemake, yadage
  - Compute backends: kubernetes
```

(note e.g. alphabetical ordering of values, note spaces after commas, note different leading text)